Framework/DataHandling/src/SetSampleMaterial.cpp

```diff
@@ -62,6 +62,10 @@ void SetSampleMaterial::init() {
   declareProperty("UnitCellVolume", EMPTY_DBL(), mustBePositive,
                   "Unit cell volume in Angstoms^3. Will be calculated from the "
                   "OrientedLattice if not supplied.");
+  declareProperty(
```

Comment:
I do have a slight concern about declaring this new parameter here. It will change the auto-generated Python function so that anyone calling it using positional arguments alone would be setting the wrong argument.

At time of writing we have about 100 uses in `v3.6` & 35 uses in `v3.5` that are not internal - http://reports.mantidproject.org/api/feature/1497 and could be affected. To be safe I think we should move the declaration to below the `ScatteringXSection` so that appears as a new positional argument.